protocol/contracts/beanstalk/silo/SiloFacet/SiloFacet.sol

```diff
@@ -58,6 +58,7 @@ contract SiloFacet is TokenSilo {
         mowSender(token) 
         returns (uint256 amount, uint256 _bdv, int96 stem)
     {
+        console.log('SiloFacet deposit');
```

Comment:
reminder to delete

Reply:
removed https://github.com/BeanstalkFarms/Beanstalk/commit/a2388afbd2eba63079cb2a21053aeb26555725fa